fe/src/components/Tab/ListingTab.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <v-app class="pl-10">
+      <v-navigation-drawer permanent>
+        <v-list dense>
+          <v-list-item-group>
+  
+            <!--Listing-->
+            <v-list-item href="#listing" @click="activeOption = 'li'" :class="{'option-active' : activeOption === 'li'}">
```

Comment:
create a function wherein you will check in switch case form the active option/tab and just call the function in class

Reply:
Just did this ma'am 🙇